### ISSUE TITLE
Cannot run helloworldoverlay from Visual studio 2013

### ISSUE THREAD

**Issue by Wesloc** [ValveSoftware/openvr#388]
Hello:

I generated the 64bit solution for MSVC12 by following the readme in samples directory.
'cmake .. -G "Visual Studio 12 2013 Win64" -DCMAKE_PREFIX_PATH=C:/Qt/5.6/msvc2013_64/lib/cmake'.

Then I try to build it from command line by 
'cmake --build . --target all --config Release'

This gave me following error.
MSBUILD : error MSB1009: Project file does not exist.
Switch: all.vcxproj

I then opened the samples/build/openvr_samples.sln in MSVC12.  I was able to build the helloworldoverylay

However, when i tried to run the debug build from the MSVC12, I got following error:

'This application failed to start because it could not find or load the Qt platform plugin "windows"'

When I 'google' this error, it appears that I need to copy the following folder to 'openvr\samples\bin\win64', which is where the helloworldoverlay.exe is. 

'C:\Qt\5.6\msvc2013_64\plugins\platforms'

However, even after I copied the platforms folder I still got the same error.

What did I do wrong?

Thank you very much,

Cheers,

**Comment by Wesloc** [comment]
I found out the earlier issues opened by aprentis .

To run helloworldoverlay.exe,  I have to delete all the qt dll in samples/bin/win64 and run windeployqt.exe.

**Comment by DayVidZhou** [comment]
I currently have the same problem, where do you find helloworldoverlay.exe?

**Comment by hanjiang08** [comment]
@DayVidZhou samples\bin\win32

**Comment by off99555** [comment]
You have the exact same problem as me.
I deleted all the QT*.dll inside the folder you suggested, ran windeployqt.exe (not sure what it does, but maybe it just generates the dll files?).
I then copied the QT*.dll from `C:\Qt\5.9.9\msvc2013_64\bin` to `samples\bin\win32` folder. I run the exe file then it works.
Here's the result:
![image](https://user-images.githubusercontent.com/15215732/95561237-a196ca80-0a44-11eb-8e79-934bb1aaeb9c.png)
It's a pretty simple overlay. It just creates another tab on the SteamVR Dashboard which shows a bunch of GUI elements.
